Calendars/Calendars.Plugin.Abstractions/ICalendars.cs

```diff
@@ -88,5 +88,15 @@ public interface ICalendars
       /// <exception cref="System.UnauthorizedAccessException">Calendar access denied</exception>
       /// <exception cref="Calendars.Plugin.Abstractions.PlatformException">Unexpected platform-specific error</exception>
       Task<bool> DeleteEventAsync(Calendar calendar, CalendarEvent calendarEvent);
+
+      /// <summary>
+      /// Adds a reminder to the specified calendar event
+      /// </summary>
+      /// <param name="calendarEvent">Event to add</param>
+      /// <param name="reminder">Reminder to add</param>
+      /// <returns>If successful</returns>
+      /// <exception cref="ArgumentException">If calendar event is not create or note valid</exception>
```

Comment:
Typos... "If calendar event is not create or note valid"...

Also "Event to add" (probably should be something like "Event to add the reminder to")